test/e2e/tests/phishing-controller/mocks.js

```diff
@@ -88,16 +96,17 @@ async function setupPhishingDetectionMocks(
     });
 
   await mockServer
-    .forGet('https://github.com/MetaMask/eth-phishing-detect/issues/new')
+    .forGet(`${C2_DOMAIN_BLOCKLIST_URL}`)
```

Comment:
NIT:
forGet can just take `C2_DOMAIN_BLOCKLIST_URL` instead of `${C2_DOMAIN_BLOCKLIST_URL}`